pkg/tests/api/prometheus/prometheus_test.go

```diff
@@ -134,7 +133,7 @@ func TestIntegrationPrometheus(t *testing.T) {
 
 		require.NotNil(t, outgoingRequest)
 		require.Equal(t, "/api/v1/query_range", outgoingRequest.URL.Path)
-		require.Contains(t, outgoingRequest.URL.String(), "&q1=1&q2=2")
+		require.Contains(t, outgoingRequest.URL.String(), "q1=1&q2=2")
```

Comment:
Maybe the following given Prometheus uses POST request?

```suggestion
		require.Contains(t, outgoingRequest.URL.String(), "?q1=1&q2=2")
```